packages/app/src/layouts/default.vue

```diff
@@ -15,7 +15,13 @@
       :page-name="currentRoute.name?.toString()"
       data-cy="app-header-bar"
       :allow-automatic-prompt-open="true"
-    />
+    >
+      <template #banner>
+        <EnableNotificationsBanner
+          v-if="query.data.value && showEnableNotificationsBanner"
```

Comment:
Could embed the `query.data.value` condition into `showEnableNotificationsBanner` - looking at it, it may already be covered since that flag relies on values nested within the query result